osf/models/node.py

```diff
@@ -1991,8 +1992,12 @@ def use_as_template(self, auth, changes=None, top_level=True):
             # TODO: explicitly define attributes which may be changed.
         except (AttributeError, KeyError):
             attributes = dict()
-
+        if self.is_deleted:
```

Comment:
Can you comment on this change? Not listed in side effects nor does it look necessary. Does it cause problems?